src/serac/physics/base_physics.cpp

```diff
@@ -363,10 +363,12 @@ std::unordered_map<std::string, FiniteElementState> BasePhysics::getCheckpointed
 
   if (checkpoint_to_disk_) {
     for (const auto& state_name : stateNames()) {
-      previous_states_map.emplace(state_name, state(state_name));
       previous_states_ptrs.emplace_back(const_cast<FiniteElementState*>(&state(state_name)));
     }
     StateManager::loadCheckpointedStates(cycle_to_load, previous_states_ptrs);
+    for (const auto& state_name : stateNames()) {
```

Comment:
Does `loadCheckpointedStates` alter the order of `stateNames`? If so can you put a comment here?

Reply:
No, it doesn't change the order.  But, it does copy the values from state(state_name).  The issue was we were copying those values when making that map, but that was happening before we read from disk.  So the map fields were out of date.

Reply:
I can see me copy/pasting this line back into the upper loop in the future.....